tests/tagged_vector.cpp

```diff
@@ -73,3 +73,13 @@ TEST(TaggedVector, Operators)
     ASSERT_EQ(4 + c, (ddc::detail::TaggedVector<int, double>(5)));
     ASSERT_EQ(4 * a, (ddc::detail::TaggedVector<int, double, float>(4, 8)));
 }
+
+TEST(TaggedVector, Conversion)
+{
+    struct Tag1;
+    struct Tag2;
+    ddc::detail::TaggedVector<int, Tag1, Tag2> a(1, 2);
+    ddc::detail::TaggedVector<double, Tag1, Tag2> b(a);
```

Comment:
```suggestion
    ddc::detail::TaggedVector<double, Tag2, Tag1> b(a);
```

Should that work also?

Reply:
Yes it should. I have added an additional test

Reply:
great, thanks :+1: